### PR TITLE
Allow types to be excluded from navigation

### PR DESCRIPTION
**Pull request by elioschmutz**
Add behavior to exclude from navigation to the following types in following templates:

Web:

ftw.simplelayout.Contentpage
ftw.news.NewsFolder

Workspace

ftw.simplelayout.Contentpage

### CODE REVIEW

File: bobtemplates/web/+package.fullname+/+package.part_1+/+package.part_2+/profiles/default/types/ftw.news.NewsFolder.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<object name="ftw.news.NewsFolder" meta_type="Dexterity FTI"
+    xmlns:i18n="http://xml.zope.org/namespaces/i18n">
+
+    <property name="behaviors">
```
Comment (mbaechtold) [inline bobtemplates/web/+package.fullname+/+package.part_1+/+package.part_2+/profiles/default/types/ftw.news.NewsFolder.xml]: Fehlt hier noch ein `purge="False"`?

File: bobtemplates/web/+package.fullname+/+package.part_1+/+package.part_2+/profiles/default/types/ftw.simplelayout.ContentPage.xml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<object name="ContentPage"
+    meta_type="Factory-based Type Information with dynamic views">
```
Comment (mbaechtold) [inline bobtemplates/web/+package.fullname+/+package.part_1+/+package.part_2+/profiles/default/types/ftw.simplelayout.ContentPage.xml]: Ist das nicht Dexterity? Siehe https://github.com/4teamwork/ftw.simplelayout/blob/master/ftw/simplelayout/contenttypes/profiles/default/types/ftw.simplelayout.ContentPage.xml#L2

File: bobtemplates/workspace/+package.fullname+/+package.part_1+/+package.part_2+/profiles/default/types/ftw.simplelayout.ContentPage.xml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<object name="ContentPage"
+    meta_type="Factory-based Type Information with dynamic views">
```
Comment (mbaechtold) [inline bobtemplates/workspace/+package.fullname+/+package.part_1+/+package.part_2+/profiles/default/types/ftw.simplelayout.ContentPage.xml]: Ist das nicht Dexterity? Siehe https://github.com/4teamwork/ftw.simplelayout/blob/master/ftw/simplelayout/contenttypes/profiles/default/types/ftw.simplelayout.ContentPage.xml#L2

File: bobtemplates/workspace/+package.fullname+/+package.part_1+/+package.part_2+/profiles/default/types/ftw.simplelayout.ContentPage.xml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<object name="ContentPage"
```
Comment (mbaechtold) [inline bobtemplates/workspace/+package.fullname+/+package.part_1+/+package.part_2+/profiles/default/types/ftw.simplelayout.ContentPage.xml]: `ContentPage`-> `ftw.simplelayout.ContentPage`

File: bobtemplates/web/+package.fullname+/+package.part_1+/+package.part_2+/profiles/default/types/ftw.simplelayout.ContentPage.xml [hunk 5]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<object name="ContentPage"
```
Comment (mbaechtold) [inline bobtemplates/web/+package.fullname+/+package.part_1+/+package.part_2+/profiles/default/types/ftw.simplelayout.ContentPage.xml]: `ContentPage`-> `ftw.simplelayout.ContentPage`

File: bobtemplates/web/+package.fullname+/+package.part_1+/+package.part_2+/profiles/default/types/ftw.news.NewsFolder.xml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<object name="ftw.news.NewsFolder" meta_type="Dexterity FTI"
+    xmlns:i18n="http://xml.zope.org/namespaces/i18n">
```
Comment (mbaechtold) [inline bobtemplates/web/+package.fullname+/+package.part_1+/+package.part_2+/profiles/default/types/ftw.news.NewsFolder.xml]: ✂️ because it is not used.

**Comment by elioschmutz** [comment]
😄  wonder why it was working. I copied this from another package!

👍  I adjusted all the things now! Thank

**Pull request merged**